.env.postgres.travis

```diff
@@ -0,0 +1,14 @@
+APP_ENV=testing
+APP_KEY=base64:NTrXToqFZJlv48dgPc+kNpc3SBt333TfDnF1mDShsBg=
+
+DB_CONNECTION=pgsql
```

Comment:
You should propose a new "pgsqltesting" connection configuration instead, using DB_TEST_* variables, like the "testing" configuration in config/database.php

Reply:
@asbiin you are welcome to take over. I'm not using monica anymore (partly because of the database issues)